STYLEGUIDE.md

```diff
@@ -32,16 +33,63 @@ The team chose Vue because of its broad usage across the development industry as
 Vue files (`.vue`) are Single-File Components that have `<template>`, `<script>` and `<style>` blocks. Conventions for writing Vue for activist include:
 
 - `<template>` blocks should come first, `<script>` second and `<style>` last
-- [TypeScript](https://www.typescriptlang.org/) should be used where ever possible within `<script>` blocks
-  - This will be more strictly enforced as the project matures
+- [TypeScript](https://www.typescriptlang.org/) should be used where ever possible within `<script>` blocks with `defineProps`
 - Self-closing components (`<Component />`) should be used for any component that doesn't have content
   - This style is [strongly recommended in Vue](https://vuejs.org/guide/essentials/component-basics.html#dom-template-parsing-caveats)
   - Generally if a component has a `<slot>` then this would imply that it would normally have content and thus require a closing tag
 
-<!-- <a id="typescript"></a>
+<a id="typescript"></a>
 
-## TypeScript -->
+## TypeScript [`⇧`](#contents)
 
+TypeScript configuration is ongoing. PR's are always welcome to improve the developer experience and project infrastructure.
+
+Currently `typescript.strict` and `typescript.typeCheck` in `nuxt.config.ts` are not enabled.
+
+> In the future, once all TS errors are resolved, activist will be enabling these strict checks. This was done to allow building the app outside `docker`. Local and Netlify builds proceed despite TS errors with strict checks disabled.
```

Comment:
Docker should be capitalized 🙃 Right? In this case because we're talking the software and not the command.